src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml

```diff
@@ -25,6 +25,19 @@
 	<SolidColorBrush x:Key="SeparatorBrush"
 					 Color="{ThemeResource MaterialOnSurfaceColor}"
 					 Opacity="0.12"/>
+
+	<!--
+		Some of the pages design flip the dichotomy of Background colors with their OnBackground complement.
+		So we have those brushes to avoid the 
+	-->
+	<SolidColorBrush x:Key="AlternateBackgroundBrush"
+					 Color="{ThemeResource MaterialOnBackgroundColor}" />
+	<SolidColorBrush x:Key="AlternateOnBackgroundBrush"
```

Comment:
Wouldn't it be better as **OnAlternate**BackgroundBrush?

Reply:
I wanted the preserve the pattern of MaterialOnBackgroundBrush and not OnMaterialBackgroundBrush
Also, it kind of mess up the alphabetical order (not issue I know but still).